service/src/main/java/bio/terra/cbas/controllers/RunsApiController.java

```diff
@@ -47,44 +50,21 @@ private Date convertToDate(OffsetDateTime submissionTimestamp) {
     return null;
   }
 
-  private RunLog convertToRunLog(WorkflowQueryResult queryResult) {
-    // Note: Cromwell's /query endpoint doesn't return 'workflowUrl' or 'workflowInputs' hence
-    // Setting it 'null' for now
-    return new RunLog()
-        .runId(queryResult.getId())
-        .state(convertToRunState(queryResult.getStatus()))
-        .workflowUrl(null)
-        .name(queryResult.getName())
-        .workflowParams(null)
-        .submissionDate(convertToDate(queryResult.getSubmission()));
-  }
+  private RunLog runToRunLog(Run run) {
 
-  @Autowired
-  public RunsApiController(CromwellServerConfiguration cromwellConfig) {
-    this.cromwellConfig = cromwellConfig;
+    return new RunLog()
+        .runId(run.id().toString())
+        .workflowUrl(run.runSet().method().methodUrl())
+        .submissionDate(convertToDate(run.submissionTimestamp()))
+        .state(convertToRunState(run.status()));
```

Comment:
This should also return workflow name (`name`) and input params definition ( `workflow_params`)